From stern@rowland.harvard.edu  Tue Jul 28 10:06:26 2009
From: Alan Stern <stern@rowland.harvard.edu>
Date: Tue, 28 Jul 2009 11:56:17 -0400 (EDT)
Subject: USB: usbtest: no need for USB_DEVICEFS
To: Greg KH <greg@kroah.com>
Message-ID: <Pine.LNX.4.44L0.0907281155190.2653-100000@iolanthe.rowland.org>


THis patch (as1270) allows the usbtest module to be built even when
USB_DEVICEFS isn't configured.  Tests can be performed without
USB_DEVICEFS, using the /dev/bus/usb/*/* device files.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/misc/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/misc/Kconfig
+++ b/drivers/usb/misc/Kconfig
@@ -220,7 +220,7 @@ config USB_IOWARRIOR
 
 config USB_TEST
 	tristate "USB testing driver"
-	depends on USB && USB_DEVICEFS
+	depends on USB
 	help
 	  This driver is for testing host controller software.  It is used
 	  with specialized device firmware for regression and stress testing,
